### PR TITLE
Replace GetString with GetTextReader for Data / Metadata

### PR DESCRIPTION
**Pull request by thefringeninja**
Same idea as #248

### CODE REVIEW

File: src/SqlStreamStore.Postgres/PostgresStreamStore.ListStreams.cs [hunk 1]
```diff
@@ -27,7 +28,9 @@ public partial class PostgresStreamStore
             using(var connection = await OpenConnection(cancellationToken))
             using(var transaction = connection.BeginTransaction())
             using(var command = GetListStreamsCommand(pattern, maxCount, afterIdInternal, transaction))
-            using(var reader = await command.ExecuteReaderAsync(cancellationToken).NotOnCapturedContext())
+            using(var reader = await command
+                .ExecuteReaderAsync(cancellationToken)
```
Comment (jeffijoe) [inline src/SqlStreamStore.Postgres/PostgresStreamStore.ListStreams.cs]: Didn't you forget `CommandBehavior.SequentialAccess` here?

Comment (thefringeninja) [reply src/SqlStreamStore.Postgres/PostgresStreamStore.ListStreams.cs]: No. Stream ids have a limited size.

Comment (jeffijoe) [reply src/SqlStreamStore.Postgres/PostgresStreamStore.ListStreams.cs]: Gotcha.

**Comment by damianh** [comment]
Valuable for metadata as well? 💭

**Pull request merged**